docs/source/containers.rst

```diff
@@ -175,9 +175,9 @@ to make a new container in your own package.
 
 Then, add an
 `entry point <https://packaging.python.org/specifications/entry-points/>`_
-specified by the **happi.containers** keyword to your package's ``setup.py``.
+specified by the **happi.containers** keyword to your package's ``pyproject.toml``.
 Example entry points can be found `here
-<https://github.com/pcdshub/pcdsdevices/blob/master/setup.py>`_.
+<https:://github.com/pcdshub/pcdsdevices/blob/master/pyproject.toml>`_.
```

Comment:
I think the extra `:` here is non-functional but it isn't technically causing any issues
```suggestion
<https://github.com/pcdshub/pcdsdevices/blob/master/pyproject.toml>`_.
```